tests/ParserTest.php

```diff
@@ -2,19 +2,9 @@
 namespace DemandwareXml\Test;
 
 use DemandwareXml\Parser;
-use DemandwareXml\Parser\{
-    AssignmentNodeParser,
-    BundleNodeParser,
-    BundleSimpleNodeParser,
-    CategoryNodeParser,
-    CategorySimpleNodeParser,
-    ProductNodeParser,
-    ProductSimpleNodeParser,
-    SetNodeParser,
-    SetSimpleNodeParser,
-    VariationNodeParser,
-    VariationSimpleNodeParser
-};
+use DemandwareXml\Parser\{AssignmentNodeParser, BundleNodeParser, BundleSimpleNodeParser, CategoryNodeParser, CategorySimpleNodeParser, ProductNodeParser, ProductSimpleNodeParser, SetNodeParser, SetSimpleNodeParser, VariationNodeParser, VariationSimpleNodeParser};
+use DemandwareXml\XmlException;
```

Comment:
This can be merged with `DemandwareXml\Parser` above.

Reply:
Done, thanks @WillJW